maltd-frontend/cypress/integration/tests/add-project.js

```diff
@@ -0,0 +1,23 @@
+/* global cy, before */
+// / <reference types="Cypress" />
```

Comment:
what does this do?

Reply:
1. Line 1 needed to be added as per the ESLint code quality to exclude cy, before cypress comments
2. Line 2 is added to initiate cypress code completion feature